new_lineage_panel/src/CustomNodes.tsx

```diff
@@ -319,12 +324,12 @@ export const TableNode: FunctionComponent<NodeProps> = ({ data }) => {
               View Details
             </div>
 
-            <div
+            {/* <div
```

Comment:
why this change?

Reply:
Have made changes for the 2nd ticket as well: [AI-1354](https://altimateai.atlassian.net/browse/AI-1354)